.github/workflows/event-consumer.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
 
       - name: Compile, test and assembly
-        run: sbt "project eventconsumer" "compile" "test" "assembly"
+        run: sbt "project commoneventconsumer" "compile" "test" "project eventconsumer" "compile" "test" "assembly"
```

Comment:
Do you mean common here? Or is `commoneventconsumer` a shared app?

Reply:
Yeah, the `commoneventconsumer` is the shared app which `eventconsumer` depends on - https://github.com/guardian/mobile-n10n/blob/main/build.sbt#L328